modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,209 @@
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+      tlsConfig:
+        ca:
+          secret:
+            name: selfsigned-mtls-bundle
+            key: ca.crt
+        cert:
+          secret:
+            name: selfsigned-mtls-bundle
+            key: client.crt
+        keySecret:
+          name: selfsigned-mtls-bundle
+          key: client.key
+----
+
+. Add write relabel configuration values after the authentication credentials:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        <endpoint_authentication_credentials>
+        <write_relabel_configs>
+----
++
+For `<write_relabel_configs>` substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
++
+The following example shows how to forward a single metric called `my_metric`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        writeRelabelConfig:
+        - source_labels: [__name__]
+          regex: 'my_metric'
+          action: keep
+
+----
++
+See the link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config[Prometheus relabel_config documentation] for information about write relabel configuration options.
+
+. If required, configure remote write for the Prometheus instance that monitors user-defined projects by changing the `name` and `namespace` `metadata` values as follows:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        <endpoint_authentication_credentials>
+        <write_relabel_configs>
+----
++
+[NOTE]
+====
+The Prometheus config map component is called `prometheusK8s` in the `cluster-monitoring-config` `ConfigMap` object and `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object.
+====
+
+. Save the file to apply the changes to the `ConfigMap` object.
+The pods affected by the new configuration restart automatically.
++
+[NOTE]
+====
+Configurations applied to the `user-workload-monitoring-config` `ConfigMap` object are not activated unless a cluster administrator has enabled monitoring for user-defined projects.
```

Comment:
`a cluster administrator`
->
`the cluster administrator`

Reply:
There can be multiple administrators no? `a cluster administrator` should be fine imho.

Reply:
I agree with @jan--f. Leaving as-is.